packages/shell-api/src/shard.ts

```diff
@@ -424,4 +426,29 @@ export default class Shard extends ShellApiWithMongoClass {
     }
     return this.stopBalancer();
   }
+
+  @returnsPromise
+  @apiVersions([])
+  @serverVersions(['6.1.0', ServerVersions.latest])
+  @returnType('AggregationCursor')
+  async getShardedDataDistribution(): Promise<AggregationCursor> {
+    this._emitShardApiCall('getShardedDataDistribution', {});
+
+    const cursor = await this._database.aggregate([{ $shardedDataDistribution: {} }]);
```

Comment:
Actually, looking at this again I have one idea:

```suggestion
  async getShardedDataDistribution(options = {}): Promise<AggregationCursor> {
    this._emitShardApiCall('getShardedDataDistribution', {});

    const cursor = await this._database.aggregate([{ $shardedDataDistribution: options }]);
```

This way, if `$shardedDataDistrubition` ever starts taking arguments, all mongosh versions that provide this helper can pass them along. How does that sound?

Reply:
Sounds great! Thanks